src/index.html

```diff
@@ -7,6 +7,11 @@
 
   <meta name="viewport" content="width=device-width, initial-scale=1">
   <link rel="icon" type="image/x-icon" href="favicon.ico">
+
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">
+  <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
```

Comment:
jquery doesnt usually play well with angular2

Reply:
oh. jquery is needed for bootstrap. its written in the documentation at
https://getbootstrap.com/

I'll comment it out and bring it in in case bootstrap css doesn't work.